package-lock.json

```diff
@@ -536,6 +536,12 @@
       "integrity": "sha512-SQaNzWQ2YZSr7FqAyPPiA3FYpux2Lqh3HWMZQk47x3xbMCqgC/w0dY3dw9rGqlweDDkrySQBcaScXWeR+Yb11Q==",
       "dev": true
     },
+    "@types/rewire": {
```

Comment:
I always advise against having this file in repo. Especially when there is not publishing configured